README.md

```diff
@@ -1,5 +1,7 @@
 # Tupaia
 
+[![Codeship Status for beyondessential/tupaia](https://app.codeship.com/projects/70159bc0-0dac-0138-fdcb-260b82737f4e/status?branch=master)](https://app.codeship.com/projects/379708)
```

Comment:
Nice!! Yeah good question... I think `dev` would be more useful to us? Add a note to that alt text though.

Reply:
I agree for `dev`, I just thought it may be more "proper" to use `master` since we've gone open source and that's what people will clone by default.

On the other hand, it's not like this repo is used as a library or something, so happy to change that to dev

Reply:
Actually, I don't think that `master` is what people will clone by default! So `dev` is best for both reasons :-)